src/Utilities/copy.jl

```diff
@@ -87,12 +87,16 @@ function copyconstraints!(dest::MOI.ModelLike, src::MOI.ModelLike, idxmap::Index
         end
     end
 
-    return passattributes!(dest, src, idxmap, cis_src)
+    return passattributes!(dest, src, copynames, idxmap, cis_src)
 end
 
 attribute_value_map(idxmap, f::MOI.AbstractFunction) = mapvariables(idxmap, f)
 attribute_value_map(idxmap, attribute_value) = attribute_value
 function defaultcopy!(dest::MOI.ModelLike, src::MOI.ModelLike)
+    warn("defaultcopy!(dest, src) is deprecated, use defaultcopy!(dest, src, false) instead or defaultcopy!(dest, src, true) if you do not want to copy names.")
```

Comment:
Are `false` and `true` swapped here? `true` is the default, and you use `false` if you do not want to copy names

Reply:
Oups, indeed